lib/venue.rb

```diff
@@ -1,2 +1,47 @@
 class Venue
+  attr_accessor :bird_type, :capacity, :capacity
+
+  def initialize(bird_type, capacity)
```

Comment:
Be sure to follow the interaction pattern here. 
```
pry(main)> venue = Venue.new('Bluebird', 4)
=> #<Venue:0x00007fed7c11dbd8 @capacity=4, @name="Bluebird", @patrons=[]>

pry(main)> venue.name
=> "Bluebird"
```
You can see that `Bluebird` is reference to a `venue name`